libs/klv/src/main/java/org/codice/alliance/libs/klv/GeometryUtility.java

```diff
@@ -85,4 +89,52 @@
         return Optional.empty();
     }
 
+    private static List<String> getAttributeStrings(Attribute attribute) {
+        return attribute.getValues()
+                .stream()
+                .filter(serializable -> serializable instanceof String)
+                .map(serializable -> (String) serializable)
+                .collect(Collectors.toList());
+    }
+
+    private static List<Coordinate> convertWktToCoordinates(List<String> points) {
+        WKTReader wktReader = new WKTReader();
+        return points.stream()
+                .map(wkt -> GeometryUtility.wktToGeometry(wkt, wktReader))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .map(Geometry::getCoordinate)
+                .collect(Collectors.toList());
+    }
+
+    private static Coordinate[] listToArray(List<Coordinate> coordinateList) {
+        return coordinateList.toArray(new Coordinate[coordinateList.size()]);
+    }
+
+    private static Geometry convertCoordinatesToGeometry(Coordinate[] coordinates) {
+        if (coordinates.length == 1) {
+            return new GeometryFactory().createPoint(coordinates[0]);
+        } else {
+            return new GeometryFactory().createLineString(coordinates);
```

Comment:
From the docs: "Creates a LineString using the given Coordinates; a null or empty array will create an empty LineString."